Lib/multiprocessing/resource_tracker.py

```diff
@@ -222,8 +222,8 @@ def main(fd):
             if rtype_cache:
                 try:
                     warnings.warn('resource_tracker: There appear to be %d '
-                                  'leaked %s objects to clean up at shutdown' %
-                                  (len(rtype_cache), rtype))
+                                  'leaked %s objects to clean up at shutdown: %s' %
+                                  (len(rtype_cache), rtype, rtype_cache))
```

Comment:
Not mandatory, but you could perhaps update this to use a f-string.

Reply:
Sure thing, added in commit 819915e1d2ebe9d9069fd72a8675498bc1873a80 (and fixed the formatting in commit 1611dd4d8c3744e14bc72b7b7b53eef51b6bde96).

I kept the warning split out into multiple lines since keeping the entire message on 1 line resulted in 156 characters on a line, which seems a bit long. I couldn't find a formal formatting guide for multiline f-strings, so I currently have it formatted as
```
                    warnings.warn(
                        f'resource_tracker: There appear to be {len(rtype_cache)} '
                        f'leaked {rtype} objects to clean up at shutdown: {rtype_cache}'
                    )
```
but I wasn't sure if I should go with the following instead:
```
                    warnings.warn(f'resource_tracker: There appear to be {len(rtype_cache)} '
                                  f'leaked {rtype} objects to clean up at shutdown: {rtype_cache}')
```

Is there a preference for these kinds of scenarios?

Reply:
I would go with the former in this case, to limit the indentation amount.

Reply:
Okay, sounds good - in that case, this part should be good to go!